pandas/tests/indexes/test_multi.py

```diff
@@ -149,13 +149,16 @@ def test_set_levels(self):
         levels = self.index.levels
         new_levels = [[lev + 'a' for lev in level] for level in levels]
 
-        def assert_matching(actual, expected):
+        def assert_matching(actual, expected, coerce_to_obj=True):
```

Comment:
call this ``check_dtype`` and simply pass onto ``assert_numpy_array_equal`` as the same parameter name.